catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/forms/SearchFormsLoader.java

```diff
@@ -0,0 +1,247 @@
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.catalog.ui.forms;
+
+import ddf.catalog.data.Metacard;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import javax.annotation.Nullable;
+import org.apache.commons.io.IOUtils;
+import org.boon.Boon;
+import org.codice.ddf.catalog.ui.forms.data.AttributeGroupMetacard;
+import org.codice.ddf.catalog.ui.forms.data.QueryTemplateMetacard;
+import org.codice.ddf.catalog.ui.forms.model.TemplateTransformer;
+import org.codice.ddf.configuration.AbsolutePathResolver;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Loads initial system template configuration from the file system so template defaults in
+ * distributions can vary independently.
+ *
+ * <p><i>This code is experimental. While it is functional and tested, it may change or be removed
+ * in a future version of the library.</i>
+ */
+public class SearchFormsLoader implements Supplier<List<Metacard>> {
+  private static final Logger LOGGER = LoggerFactory.getLogger(SearchFormsLoader.class);
+
+  private static final Function<File, Consumer<? super Object>> UNEXPECTED_CONFIG_CONSUMER_FACTORY =
+      file ->
+          obj -> {
+            if (!Map.class.isInstance(obj)) {
+              LOGGER.warn(
+                  "Unexpected configuration in {}, values should be maps not {}",
+                  file.getName(),
+                  obj.getClass().getName());
+            }
+          };
+
+  private static final File DEFAULT_FORMS_DIRECTORY =
+      new File(new AbsolutePathResolver("etc/forms").getPath());
+
+  private static final String FORMS_FILE_NAME = "forms.json";
+
+  private static final String RESULTS_FILE_NAME = "results.json";
+
+  private final File configDirectory;
+
+  public static Supplier<List<Metacard>> config() {
+    return new SearchFormsLoader();
+  }
+
+  public SearchFormsLoader() {
+    this(DEFAULT_FORMS_DIRECTORY);
+  }
+
+  public SearchFormsLoader(File configDirectory) {
+    this.configDirectory = configDirectory;
+  }
+
+  @Override
+  public List<Metacard> get() {
+    if (!configDirectory.exists()) {
+      LOGGER.warn("Could not locate forms directory [{}]", configDirectory.getAbsolutePath());
+      return Collections.emptyList();
+    }
+
+    // What's our policy here? Do we want the system to blowup / make noise if this isn't possible?
+    // Especially due to the security manager?
```

Comment:
Do we need this comment still?  Seems reasonable to log and return an empty list in this case.

Reply:
I'm going to nuke this check. I'm concerned that a failure to read will be gracefully handled but if the SM blows up I don't think we want that to occur quietly. 

This is also pointless after the `exists()` check anyway. The SM will throw an exception in the `exists()` before we even get here.